test/e2e/helm/kuma_helm_upgrade.go

```diff
@@ -68,18 +74,17 @@ func UpgradingWithHelmChart() {
 
 			k8sCluster := cluster.(*K8sCluster)
 
-			upgradeOptsFuncs := []DeployOptionsFunc{
-				WithHelmReleaseName(releaseName),
-			}
+			upgradeOptsFuncs := append(KumaK8sDeployOpts,
+				WithHelmReleaseName(releaseName))
 
 			err = k8sCluster.UpgradeKuma(core.Standalone, upgradeOptsFuncs...)
 			Expect(err).ToNot(HaveOccurred())
 		},
-		Entry("should successfully upgrade from chart v0.4.4", testCase{
-			initialChartVersion: "0.4.4",
+		Entry("should successfully upgrade from chart v"+OlderChart, testCase{
+			initialChartVersion: OlderChart,
 		}),
-		Entry("should successfully upgrade from chart v0.4.5", testCase{
-			initialChartVersion: "0.4.5",
+		Entry("should successfully upgrade from chart v"+OldChart, testCase{
+			initialChartVersion: OldChart,
```

Comment:
to be honest - no need to test two charts here. Let's just check one.
We should test what is the oldest version that we can upgrade to.